src/index.ts

```diff
@@ -1,59 +1,33 @@
-import express, { Request, Response } from 'express';
-import { serve } from 'swagger-ui-express'
-import swagger from './swagger';
+import express, { Request, Response } from "express";
+const { MongoClient } = require("mongodb");
 
+import swaggerUi from "swagger-ui-express";
+import swaggerJSDoc from "swagger-jsdoc";
+import swagger from "./swagger";
+import routes from "./routes";
+import mongoose from "mongoose";
+import bodyParser from 'body-parser'
 const app = express();
 const PORT: number = 3000;
-/**
- * @swagger
- * /api-docs
- */
-app.use('/api-docs', serve, swagger());
-app.get('/', (req: Request, res: Response) => {
-    res.send('Welcome to the root path!');
-});
-/**
- * @swagger
- * /health-check:
- *   get:
- *     summary: Check if the server is running
- *     responses:
- *       200:
- *         description: Server is running
- */
-app.get('/health-check', (req: Request, res: Response) => {
-    res.json({ status: 'Server is running!' });
+
+const uri = "mongodb+srv://hrunushunasophia13:K2kWiIHQwrAvRzkO@cluster0.cev8jny.mongodb.net/?retryWrites=true&w=majority";
```

Comment:
bad idea to commit suck sensitive data to github repository. Please use .env for it.